modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommercePaymentTerms.testcase

```diff
@@ -25,14 +25,51 @@ definition {
 		CommerceConfiguration.commerceInitializersTearDown();
 	}
 
-	@description = "[Please Update]"
+	@description = "Add a new payment term to Terms and Conditions."
 	@ignore = "Test Stub"
 	@priority = "4"
 	test AddANewPaymentTerm {
 		property portal.acceptance = "false";
 
 		// TODO COMMERCE-8279 AddANewPaymentTerm pending implementation
 
+		task ("Given a terms and coditions page.") {
```

Comment:
This would be better as "Given the Sales Manager goes into the Terms and Conditions" page.